ora2/jobs/SetOra2VersionOnSandbox.groovy

```diff
@@ -0,0 +1,73 @@
+/*
+    Variables consumed from the EXTRA_VARS input to your seed job in addition
+    to those listed in the seed job.
+
+    JOB_FOLDER_NAME: some-folder
+    CONTACT_EMAIL : e-mail-contact@example.com
+
+    The following parameters are needed for the Runansible Job below
+
+    SSH_AGENT_KEY: dictionary of key per environment
+    CONFIGURATION_REPO: standard default available
+    CONFIGURATION_BRANCH: defaults to master
+    ACCESS_CONTROL: list of Jenkins matrix targets who should get read access (edx*team or username)
+    SCHEDULE: Cron syntax consumed by Jenkins triggered builds
+    SSH_USER: ansible connect user - defaults to ubuntu.  Make sure this matches up with SSH_AGENT_KEY
+    CONTACT_EMAIL: email address to be notified when this job fails
+    PLAYBOOK:  The playbook used in this job
+*/
+
+package ora2.jobs
+
+import static devops.jobs.RunRemoteAnsiblePlaybook.job as RunRemoteAnsiblePlaybookJob
+class SetOra2VersionOnSandbox {
+
+    public static def job = { dslFactory, extraVars ->
+        def jobRoot = extraVars.get("JOB_FOLDER_NAME")
+
+        def jobName = 'SetOra2VersionOnSandbox'
+        if (jobRoot != null && jobRoot != '') {
+            jobName = jobRoot + '/SetOra2VersionOnSandbox'
+        }
+        return dslFactory.job(jobName) {
```

Comment:
So I think you'll probably always want to have a folder name specified. Also, I used a get with a default, and I think the way I have it might look a bit cleaner? 

`dslFactory.job(extraVars.get('JOB_FOLDER_NAME', 'ora2') + '/some-job-name')`

Let me know what your opinion, but I think we should be consistent with each other.